### ISSUE TITLE
Django 1.10 compatibility: RemovedInDjango110Warning

### ISSUE THREAD

**Issue by PetrDlouhy** [adamcharnock/django-su#43]
There is warning about future incompatibility with Django 1.10
```
/home/petr/soubory/programovani/Auto-mat/DPNK/dpnk/env/lib/python3.4/site-packages/django_su/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
```

**Comment by bashu** [comment]
What version? Have you tried ``develop`` branch?

**Comment by PetrDlouhy** [comment]
I used PyPI package, but I looked at Travis tests and didn't see any Django 1.10 tests.
You can test development version with `allow_features`. If you did that, you would realize, that you foregotten to remove `patterns` import.

Both these changes are in PR #44

**Comment by bashu** [comment]
closed via @4c060a84c9bce350cbd62c85f9747eabd74bc48d
